newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js

```diff
@@ -72,22 +70,14 @@ const homePageMenuTabs: { [tab: string]: HomePageMenuTab } = {
       <SunIcon fontSize={fontSize} color={color} />
     ),
   },
-  build: {
-    label: <Trans>Build</Trans>,
-    tab: 'build',
-    id: 'home-build-tab',
+  create: {
+    label: <Trans>Create</Trans>,
+    tab: 'create',
+    id: 'home-create-tab',
```

Comment:
This makes me think that in app tutorials will need to be updated

Reply:
I can only see the fling game to update no?
the others don't go back to the homepage

Reply:
Most def